control-plane/subcommand/server-acl-init/command_test.go

```diff
@@ -125,7 +124,7 @@ func TestRun_Defaults(t *testing.T) {
 		"-resource-prefix=" + resourcePrefix,
 	}
 	responseCode := cmd.Run(args)
-	require.Equal(0, responseCode, ui.ErrorWriter.String())
+	require.Equal(t, 0, responseCode, ui.ErrorWriter.String())
```

Comment:
updated these tests to no longer use the `require := require.New(t)` pattern and use the `require.xyz(t, ...)` pattern